db/schema.rb

```diff
@@ -126,10 +126,14 @@
   create_table "user_settings", force: :cascade, options: "ENGINE=InnoDB DEFAULT CHARSET=utf8" do |t|
```

Comment:
**Reporter**: CHECKSTYLE
**Rule**: com.puppycrawl.tools.checkstyle.Metrics/LineLength
**Severity**: INFO
**File**: db/schema.rb L126

Line is too long. [101/80]

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>*